### PR TITLE
Deuglify 

### PR DESCRIPTION
**Pull request by grafixeyehero**
Please make sure that I didn't miss something

### CODE REVIEW

File: src/controllers/addpluginpage.js [hunk 1]
```diff
@@ -1,155 +1,239 @@
-define(["jQuery", "loading", "libraryMenu", "globalize", "connectionManager", "emby-button"], function($, loading, libraryMenu, globalize, connectionManager) {
-    "use strict";
+define(['jQuery', 'loading', 'libraryMenu', 'globalize', 'connectionManager', 'emby-button'], function ($, loading, libraryMenu, globalize, connectionManager) {
+    'use strict';
 
     function populateHistory(packageInfo, page) {
-        var html = "";
-        var length = Math.min(packageInfo.versions.length, 10);
-        for (var i = 0; i < length; i++) {
+
```
Comment (cvium) [inline src/controllers/addpluginpage.js]: There's a lot of unnecessary newlines.

Comment (grafixeyehero) [reply src/controllers/addpluginpage.js]: https://github.com/jellyfin/jellyfin-web/pull/395/commits/c1c9d2d5055bacfc26ece03943fb21830153b444

**Comment by dkanada** [comment]
@grafixeyehero are you using `eslint --fix` or a different tool to deuglify these files? I would keep the double quotes and also avoid so many newlines in all the modified files.

**Comment by grafixeyehero** [comment]
@dkanada no i didn't use the above tool, but i use the old versions by matching the code

**Comment by grafixeyehero** [comment]
closing the pull requests for know. 
@dkanada commited harder to review since they're so large.
 I will gone to reduce it

**Pull request closed without merging**